framework/src/st_things/things_stack/framework/things_req_handler.c

```diff
@@ -197,17 +197,12 @@ static OCEntityHandlerResult convert_ap_infor(things_resource_s *target_resource
 	} else {
 		rep = target_resource->rep;
 	}
+
+	THINGS_LOG_D(TAG, "Create response for accesspoints resource, WiFi list count is: %d", list_cnt);
 	things_representation_s **child_rep = (things_representation_s **)things_malloc(list_cnt * sizeof(things_representation_s *));
 	access_point_info_s *wifi_scan_iter = p_list;
 	int iter;
 	for (iter = 0; iter < list_cnt; ++iter) {
-#ifdef CONFIG_DEBUG_ST_THINGS_DEBUG
-		THINGS_LOG_V(TAG, "e_ssid : %s", wifi_scan_iter->e_ssid);
-		THINGS_LOG_V(TAG, "signal_level : %s", wifi_scan_iter->signal_level);
```

Comment:
Would not it be better to put the rssi value at the debug level?

Reply:
This log doesn't need. It spend some time to pring log.

Reply:
This is the only logic that runs in DEBUG mode.
In my opinion, debugging mode, it does not seem to have much relation to performance.
I think need more information for debugging (rssi signal) than the time spent in print

Reply:
When enable ST_THIGNS_DEBUG option. 
Those log printing logic will  over a  seconds only for two printing logics about scanning result and creating response. 

Actually in last performance check, I know something that If DEBUG was enabled  the easy-setup time will increase more 20s. And It will increase spending time if there is ap device more. 

And now we can get ap list and infomations during do easy-setup. ST APP(since 1.7.33-26) will support this function and user must select ap to use.